.github/workflows/cuda-gpu-ci.yml

```diff
@@ -25,6 +29,10 @@ jobs:
           git fetch origin +refs/pull/${{ inputs.pr_id }}/head:pr-${{ inputs.pr_id }}
           git checkout pr-${{ inputs.pr_id }}
           echo "Checked out commit $(git rev-parse HEAD)"
+      - name: Store commit SHA
```

Comment:
I think we need this because the commit SHA that we test/checkout via our custom "fetch a PR" is not already available as an environment variable.